vxquery-xtest/src/test/resources/Queries/XQuery/Json/Functions/general_comparison.xq

```diff
@@ -0,0 +1,28 @@
+(: Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+   
+     http://www.apache.org/licenses/LICENSE-2.0
+   
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License. :)
+
+(: JSONiq Changes to value comparison semantics :)
+(null, 2) = (1, 3),
+(null, 2) != (1, 3),
+(null, 2) < (1, 3),
+(null, 2) <= (1, 3),
+(null, 1) > (1, 3),
+(null, 0) >= (1, 3),
+null < 1,
```

Comment:
Please include all possible combinations for general comparison. (Also update the comment)